app/config/config_test.go

```diff
@@ -43,3 +43,13 @@ func TestLoadConfig(t *testing.T) {
 	// test env
 	assert.EqualValues(999, conf.Server.WriteTimeout)
 }
+
+func TestLoadConfig_Concurrent(t *testing.T) {
+	assert := assertlib.New(t)
+	assert.NotPanics(func() {
+		_, _ = Load()
```

Comment:
This line is probably unnecessary.

Reply:
it is not (to make it fail when it was failing) but don't ask me why 😁